app/controllers/users_controller.rb

```diff
@@ -1,3 +1,30 @@
 class UsersController < ApplicationController
-  def new; end
+  def show
+    @user = User.find_by id: params[:id]
+    return if @user
```

Comment:
chỗ này có thể viết như này cho gọn nhé, xóa line 3 đi 
```suggestion
    return if @user = User.find_by(id: params[:id])
```